### PR TITLE
no links in code

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

**Comment by SimonCropp** [comment]
@indualagarsamy @andreasohlund please review

**Comment by andreasohlund** [comment]
:+1:

File: packaging/nuget/nservicebus.host/EndpointConfig.cs [hunk 1]
```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
...
     public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(BusConfiguration configuration)
         {
-            // NServiceBus provides the following durable storage options
-            // To use RavenDB, install-package NServiceBus.RavenDB and then use configuration.UsePersistence<RavenDBPersistence>();
-            // To use SQLServer, install-package NServiceBus.NHibernate and then use configuration.UsePersistence<NHibernatePersistence>();
+            // NServiceBus provides multiple durable storage options
             
             // If you don't need a durable storage you can also use, configuration.UsePersistence<InMemoryPersistence>();
```
Comment (andreasohlund) [inline packaging/nuget/nservicebus.host/EndpointConfig.cs]: Remove the new lines? (https://github.com/Particular/NServiceBus.Host/blob/fixInjectedCode/packaging/nuget/nservicebus.host/EndpointConfig.cs , line 11 +13)

File: packaging/nuget/nservicebus.host/EndpointConfig.cs [hunk 2]
```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
...
-		can be found here: http://particular.net/articles/the-nservicebus-host
-	*/
     public class EndpointConfig : IConfigureThisEndpoint
     {
         public void Customize(BusConfiguration configuration)
         {
-            // NServiceBus provides the following durable storage options
-            // To use RavenDB, install-package NServiceBus.RavenDB and then use configuration.UsePersistence<RavenDBPersistence>();
-            // To use SQLServer, install-package NServiceBus.NHibernate and then use configuration.UsePersistence<NHibernatePersistence>();
+            // NServiceBus provides multiple durable storage options
```
Comment (indualagarsamy) [inline packaging/nuget/nservicebus.host/EndpointConfig.cs]: @SimonCropp - I had specifically added RavenDB and SqlServer. Telling the user we have multiple storage options but not tell them anything serves no purpose.

Instead, how about:
NServiceBus provides multiple durable storage options like RavenDB, SQLServer. Download the appropriate NuGet package and call the configuration.UsePersistence<T> API.

File: packaging/nuget/nservicebus.host/EndpointConfig.cs [hunk 3]
```diff
@@ -3,23 +3,15 @@ namespace rootnamespace
...
             
             // If you don't need a durable storage you can also use, configuration.UsePersistence<InMemoryPersistence>();
-            // more details on persistence can be found here: http://docs.particular.net/nservicebus/persistence-in-nservicebus
             
-            //Also note that you can mix and match storages to fit you specific needs. 
-            //http://docs.particular.net/nservicebus/persistence-order
+            //Also note that you can mix and match multiple storages to fit you specific needs.
```
Comment (indualagarsamy) [inline packaging/nuget/nservicebus.host/EndpointConfig.cs]: Is this note even relevant? I think mixing and matching multiple storage options should be part of the documentation and along with it what the downsides are. Let's remove it.

**Comment by andreasohlund** [comment]
@SimonCropp @indualagarsamy removed the note and reworded slightly. Please review

**Comment by indualagarsamy** [comment]
@andreasohlund - Searching for `durable storage` or `durable persistence` don't lead the user to the right topic, which is http://docs.particular.net/nservicebus/persistence/ (which needs content work as well).
 
@SimonCropp - Is there a way we can bump up that link when the user searches persistence? 

How about:
NServiceBus provides multiple durable storage options like SQLServer, etc. Refer to the documentation for the available options on persistence for more details.

**Comment by kbaley** [comment]
Or:
NServiceBus provides multiple durable storage options, including SQL Server, RavenDB, and Azure Storage Persistence. Refer to the documentation for each option for more details.

**Comment by indualagarsamy** [comment]
:+1: to what @kbaley suggested.

**Comment by andreasohlund** [comment]
Let's go with @kbaley 's suggestion?

**Comment by danielmarbach** [comment]
Ping @SimonCropp

**Pull request merged**